tests/foreman/ui/test_host.py

```diff
@@ -412,3 +412,95 @@ def test_positive_update_with_user(self):
 
         @Status: Manual
         """
+
+    @stubbed()
+    @tier3
+    def test_positive_provision_atomic_host(self):
+        """Provision an atomic host on libvirt and register it with server
+
+        @Feature: Atomic Host
+
+        @Assert: Atomic host should be provisioned and listed under
+        content-hosts/Hosts
+
+        @Status: Manual
+        """
+
+    @stubbed()
+    @tier3
+    def test_positive_register_pre_installed_atomic_host(self):
+        """Register a pre-installed atomic host with server using admin
+        credentials
+
+        @Feature: Atomic Host
+
+        @Assert: Atomic host should be registered successfully and listed under
+        content-hosts/Hosts
+
+        @Status: Manual
+        """
+
+    @stubbed()
+    @tier3
+    def test_positive_register_pre_installed_atomic_host_using_ak(self):
+        """Register a pre-installed atomic host with server using activation
+        key
+
+        @Feature: Atomic Host
+
+        @Assert: Atomic host should be registered successfully and listed under
+        content-hosts/Hosts
+
+        @Status: Manual
+        """
+
+    @stubbed()
+    @tier3
+    def test_positive_delete_atomic_host(self):
+        """Delete a provisioned atomic host
+
+        @Feature: Atomic Host
+
+        @Assert: Atomic host should be deleted successfully and shouldn't be
+        listed under hosts/content-hosts
+
+        @Status: Manual
+        """
+
+    @stubbed()
+    @tier3
+    def test_positive_bulk_delete_atomic_host(self):
+        """Delete a multiple atomic hosts
+
+        @Feature: Atomic Host
+
+        @Assert: All selected atomic hosts should be deleted successfully
+
+        @Status: Manual
+        """
+
+    @stubbed()
+    @tier3
+    def test_positive_update_atomic_host_cv(self):
+        """Update atomic-host with a new environment and content-view
+
+        @Feature: Atomic Host
+
+        @Assert: Atomic host should be updated with new content-view
+
+        @Status: Manual
+        """
+
+    @stubbed()
+    @tier3
+    def test_positive_execute_cmd_on_atomic_host_with_job_templates(self):
```

Comment:
Yeah.. I was thinking to use same names. But we may have host (content-host) specific tests which are not related to `atomic_host`. So I added `atomic_host` in test names just to differentiate between content-host and atomic.